frontend/src/metabase/visualizations/Progress.jsx

```diff
@@ -127,7 +132,7 @@ export default class Progress extends Component {
                             }}
                         />
                     </div>
-                    <div className="relative" style={{
+                    <div ref="bar" className="relative" style={{
                         backgroundColor: restColor,
                         borderRadius: BORDER_RADIUS,
                         height: "25%",
```

Comment:
If we have to set the height via refs I'd prefer not to set a height in the style object and just do it in one place.